extensions/amp-date-picker/0.1/dates-list.js

```diff
@@ -15,15 +15,17 @@
  */
 
 import {requireExternal} from '../../../src/module';
+import {rrulestr} from 'rrule/dist/esm/src/rrulestr.js';
```

Comment:
Now that https://github.com/ampproject/amphtml/pull/26123 is merged, this can just be:

```suggestion
import {rrulestr} from 'rrule';
```